src/drivers/imx/interrupt.c

```diff
@@ -70,54 +70,24 @@
  * usage in this driver.
  */
```

Comment:
@paulstelian97 really appreciate if you can add two lines about the differences between irqsteer on 8/8X vs 8M

Reply:
Later edit: notice that you can read this from the comments inside code.